pkg/build/docker_go.go

```diff
@@ -134,7 +134,7 @@ func (b *DockerGoBuilder) Build(ctx context.Context, in *api.BuildInput, ow *rpc
 
 	var (
 		basesrc = in.UnpackedSources.BaseDir
-		plansrc = in.UnpackedSources.PlanDir
+		planDir = in.UnpackedSources.PlanDir
 		sdksrc  = in.UnpackedSources.SDKDir
```

Comment:
The diff in naming convention struck me here at first. I got what's going on when I kept reading though.

Reply:
That's a fair point, I'll use go convention and rename these to `baseSrc`, `planDir`, `planSrc`, `sdkSrc`

Reply:
Done